api/Dockerfile

```
@@ -0,0 +1,34 @@
+FROM openjdk:21-jdk-slim AS build
+
+RUN apt-get update && apt-get install -y dos2unix
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)